subjects/matrix_multiplication/README.md

```diff
@@ -0,0 +1,53 @@
+## matrix_multiplication
+
+### Instructions
+
+- Define a `struct` named `Matrix` as a tuple of 2 tuples. The nested tuple will contain 2 `i32`s.
+
+- Create a **function** named `multiply` that receives a Matrix and an i32 and returns the Matrix with each number multiplied by the second argument. 
+```rust
+pub fn multiply(m: Matrix, val: i32) -> Matrix {
+}
+```
+
+`Matrix` must implement `Debug`, `PartialEq` and `Eq`. You can use `derive`.
+
+> Remember that you are defining a library, so any element that can be called from an external crate must be made public.
+
+### Usage
+
+Here is a possible program to test your function
+
+```rust
+use matrix_multiplication::multiply;
+use matrix_multiplication::Matrix;
```

Comment:
When having more than one element to be included from an external crate you may use `use matrix_multiplication::*;`.
This is also valid for your `tests` in `rust-tests`